### PR TITLE
Added WSL2 instructions

### PR DESCRIPTION
**Pull request by yostane**
- Detailed limitations of WSL1: debug and run the Swift REPL.
- Added instructions for WSL2 that allow to debug and REPL

### CODE REVIEW

File: docs/Windows.md [hunk 1]
```diff
@@ -10,6 +10,11 @@ Currently there are two supported ways to build Swift for Windows.
...
 
-## Windows Subsystem for Linux
+## Windows Subsystem for Linux (WSL)
 
 [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/about) is an Ubuntu environment.  Follow the instructions for building on [Linux](../README.md#linux)
+
+There two versions of WSL as of november 23, 2019:
+
+- WSL1: is the current stable version. Swift compilation and execution works but REPL and debugging (LLDB) hangs on startup.
+- WSL2: Both REPL and debugging work in this verion. WSL2 is still in development and is available by downloading an insider build. Fortunately, installing WSL2 is ppretty simple (switch to insider, download an insider build and run some scripts). WSL2 can be installed [by following this link](https://docs.microsoft.com/windows/wsl/wsl2-install).
```
Comment (compnerd) [inline docs/Windows.md]: typo: ppretty -> pretty

Comment (yostane) [reply docs/Windows.md]: Thanks. I fixed the typos.

**Comment by yostane** [comment]
Hi. Is there anything else I can do to complete this PR ? 
Thanks :)

**Comment by compnerd** [comment]
> Hi. Is there anything else I can do in order to complete this PR ?
> Thanks :)

The tree is currently locked due to the holidays, so we will have to wait until after the holidays.

**Comment by compnerd** [comment]
@swift-ci please smoke test

**Pull request merged**